nservicebus/hosting/cloud-services-host/shared-hosting.md

```diff
@@ -50,59 +50,33 @@ In this worker role one needs to reference the assembly that contains the Azure
-	</LocalResources>
-</WorkerRole>
-```
+Snippet:LocalResource
 
 Other configuration settings are available as well if one needs a more fine grained control on how the host works:
 
- * `DynamicHostControllerConfig.Container`: The container where the endpoint packages are stored in the storage account, defaults to `endpoints`
- * `DynamicHostControllerConfig.AutoUpdate`: Turn auto update on or off, defaults to true. Note that if it is set to false, then the host needs a reboot to pick new endpoints or versions of endpoints.
- * `DynamicHostControllerConfig.UpdateInterval`: The time between checks if updates are available, in milliseconds, defaults to `600000`.
- * `DynamicHostControllerConfig.LocalResource`: The name of the local storage resource where the zip archives will be extracted, defaults to `endpoints`
- * `DynamicHostControllerConfig.TimeToWaitUntilProcessIsKilled`: When updating an endpoint to a new version, the host will kill the current process. Sometimes this fails or takes a very long time. This property specifies how long the host should wait, if this time elapses without the process going down, the host will reboot the machine (by throwing an exception). Default value: `10000`.
- * `DynamicHostControllerConfig.RecycleRoleOnError`: By default Azure role instances will reboot when an exception is thrown from the role entrypoint, but not when thrown from a child process. If one wants the role instance to reboot in this case as well, set `RecycleRoleOnError` on true. Then the host will start monitoring the child process for errors and request a recycle when it throws.
+ * `ConnectionString`: The connectionstring to the storage account containing the endpoint packages.
+ * `Container`: The container where the endpoint packages are stored in the storage account, defaults to `endpoints`
+ * `AutoUpdate`: Turn auto update on or off, defaults to true. Note that if it is set to false, then the host needs a reboot to pick new endpoints or versions of endpoints.
+ * `UpdateInterval`: The time between checks if updates are available, in milliseconds, defaults to `600000`.
+ * `LocalResource`: The name of the local storage resource where the zip archives will be extracted, defaults to `endpoints`
+ * `TimeToWaitUntilProcessIsKilled`: When updating an endpoint to a new version, the host will kill the current process. Sometimes this fails or takes a very long time. This property specifies how long the host should wait, if this time elapses without the process going down, the host will reboot the machine (by throwing an exception). Default value: `10000`.
+ * `RecycleRoleOnError`: By default Azure role instances will reboot when an exception is thrown from the role entrypoint, but not when thrown from a child process. If one wants the role instance to reboot in this case as well, set `RecycleRoleOnError` on true. Then the host will start monitoring the child process for errors and request a recycle when it throws.
```

Comment:
Do note that those signs are culture specific. If a european reads that, he'll think it's 600...